COMMPLAN.md

```diff
@@ -0,0 +1,204 @@
+Don't use Zulip for communication that is meant for the
+*entire user community* (e.g., all of Pangeo), and not just NCAR staff.
+In those cases, I recommend using the Pangeo communication channels, as
+described in the [PANGEO Guide](PANGEO.md).
+
+### The Xdev Blog
+
+#### Why should we use it?
+
+Our blog (https://ncar.github.io/xdev) is an excellent way for us to
+communicate complex ideas, or to make announcements, to the *broader*
+community (i.e., beyond NCAR) in an easy-to-access form.  Blogs persist
+and are designed to be referenceable at a later date, which makes them
+excellent for sharing our individual (or Team) accomplishments in a
+way that can be cited.  By hosting our *own* blog, which is connected
+to Google Analytics, we can gather metrics and visit statistics on
+our blog posts, which can provide us useful information about who is
+making use of our posts.  Career-wise, blog posts are strongly
+encouraged in the software engineering community!
+
+#### What should we use it for?
+
+Use our blog as a way of publicizing new software package, new software
+versions, solutions to hard problems, or any general accomplishment.
+Use our blog to make *general public announcements* about things like
+tutorials or new services.  Use our blog to publicize Xdev Team work and
+achievements.
+
+#### What should we *not* use it for?
+
+Don't use the blog for rants or making complaints about anything.  The
+blog should only be used for *constructive* communication.
+
+### Twitter (@NCARXdev)
+
+#### Why we should use it?
+
+Twitter is a commonly used platform by many people in the Tech Industry,
+especially software engineers and scientists.  For example, a very large
+fraction of the Pangeo community regularly tweet.  Hence, Twitter makes
+an excellent platform for getting the attention of the general public
+(or the fraction of the general public interested in NCAR's Python
+work).
+
+#### What should we use it for?
+
+The Xdev Twitter account should be for making *announcements* to the
+general public, such as making an announcement of a new blog post.
+In general, Twitter *could* be used for any announcement, but because
+of the limitations of the platform (i.e., limiting the length of the
+tweet) it is best to tie the tweet to *another* communication platform,
+such as a blog post.
+
+#### What should we *not* use it for?
+
+The Xdev Twitter account should *not* be used for personal use.  Currently,
+only Kevin has the "keys" to the Twitter account.  If you want to make a
+broad announcement, contact Kevin via Zulip.
+
+### Email
+
+#### Why should we use it?
+
+Not sure.  I'm open to discussion on this, but I am beginning to feel like
+email (especially the `xdev@ucar.edu` address) should not be used by any
+of us, unless people explicitly reach out to us.  That is, I think email
+should be a way for people *outside* of NCAR to directly communicate with
+*us*.
+
+At NCAR, email is used substantially for inter-NCAR communication or for
```

Comment:
I'm not sure what you mean by "inter-NCAR communication" -- should this be "intra-NCAR"? Or "inter-lab communication within NCAR"?

Reply:
Good point.  I meant inter-lab, but I think it's better to just say intra-NCAR.

Reply:
Fixed.